ns-3.32/scratch/webrtc_test/webrtc-static.cc

```diff
@@ -23,6 +23,48 @@ const uint32_t TOPO_DEFAULT_BW     = 3000000;
 const uint32_t TOPO_DEFAULT_PDELAY =100;
 const uint32_t TOPO_DEFAULT_QDELAY =300;
 const uint32_t DEFAULT_PACKET_SIZE = 1000;
+const static uint32_t RATE_ARRAY[]= { 3000000 };
+
+class ChangeBw
```

Comment:
Please add some comments to explain how ChangeBw works.

Reply:
For example, you can say  it will pick a bandwidth from `RATE_ARRY` as the link bandwidth with one second round robin algorithm.